tests/framework/test_values.py

```diff
@@ -86,3 +87,35 @@ def test_rescale_postprocessor(manager):
     odds = value.iloc[lambda x: x.index % 2 == 1]
     assert np.all(evens == from_yearly(0.5, pd.Timedelta(days=3)))
     assert np.all(odds == from_yearly(0.5, pd.Timedelta(days=5)))
+
+
+def test_step_size_post_processor(manager):
```

Comment:
If we move this post processor inside the clock as I suggested above, this test should move inside `test_time.py`